src/logic/PlayerController.cpp

```diff
@@ -380,7 +380,7 @@ void PlayerController::unequipItem(Daedalus::GameState::ItemHandle item)
 
     EModelNode node = EModelNode::None;
 
-    if((itemData.flags & Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE) != Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE)
+    if((itemData.flags & Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE) != 0)
```

Comment:
Yes, you're right it's wrong. I didn't notice the mistake because it worked but it was just a coincidence. For checking if an item is equipable one should use mainFlags instead of flags because the former contains information about category.

Thanks for pointing it out.